src/renderer/store/helpers.ts

```diff
@@ -33,6 +33,15 @@ export interface UndoStack {
   index: number; // Used for supporting redo
 }
 
+/**
+ * All input/output user actions states
+ * Import / Export progress states
+ */
+export interface IO {
```

Comment:
Maybe call this ExportIO or similar as we may want other IO state later

Reply:
Yep, sure. I actually named it IO to allow for future IO states. But I guess we can always rename it back if we decide to go with that.

Reply:
Done.

Reply:
Ah I see what you mean - I had imagined it as we would add new root level state for other IO stuff, rather than reusing the IO state. I suppose either are fine